baseball.py

```diff
@@ -0,0 +1,47 @@
+
+from pyspark import StorageLevel
+from pyspark.ml.feature import CountVectorizer
+from pyspark.sql import SparkSession
+
+# Setup Spark
+spark = SparkSession.builder.master("local[*]").getOrCreate()
+database = "baseball"
+port = "3306"
+user = "root"
+password = "root"
+query = "select * from batter_counts"
+
+df = spark.read.format("jdbc").options(
+    url=f"jdbc:mysql://localhost:{port}/{database}",
+    driver="com.mysql.cj.jdbc.Driver",
+    query = query,
+    user=user,
+    password=password).load()
+
+# calculate the historical batting average
+df.createOrReplaceTempView("avg")
+df.persist(StorageLevel.DISK_ONLY)
+result = spark.sql("select batter, sum(Hit) as TH,sum(atBat) as TB, round(SUM(Hit)/SUM(atBat),3) as AVG \
```

Comment:
I know you're still working on rolling and you started with historic to make it easier. But editing this SQL code with the rolling code from the previous assignment should get you the rolling averages pretty easily.